charts/linkerd2/templates/prometheus.yaml

```diff
@@ -195,6 +203,11 @@ spec:
         - --storage.tsdb.retention.time=6h
         - --config.file=/etc/prometheus/prometheus.yml
         - --log.level={{lower .Values.prometheusLogLevel}}
+        {{- range $key, $value := .Values.prometheusExtraArgs}}
+        {{- if $value }}
```

Comment:
I believe boolean flags like `--web.enable-admin-api` don't require a value, so I think this `if` belongs below, something like
```
- --{{ $key }}{{ if $value }}={{ $value }}{{ end }}
```

Reply:
done!